### PR TITLE
DevTools 97 > Device mode

### PR DESCRIPTION
**Pull request by sofiayem**
- Refreshed "https://developer.chrome.com/docs/devtools/device-mode/".
- Updated screenshots.
- Documented https://developer.chrome.com/en/blog/new-in-devtools-97/#device.

### CODE REVIEW

File: site/en/docs/devtools/device-mode/index.md [hunk 1]
```diff
@@ -37,47 +38,38 @@ actually runs on a mobile device.
...
+{% Img src="image/admin/9FiBHFCzfPgP8sy6LMx7.png", alt="Toggle Device Toolbar", width="20", height="22" %} to open
 the UI that enables you to simulate a mobile viewport.
 
-{% Img src="image/admin/9Hw1ojc4jyUdo0ycwH7X.png", alt="The Device Toolbar.", width="800", height="645" %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/CkPWy2neRUswovjY1ql2.png", alt="The Device Toolbar.", width="800", height="562" %}
 
-**Figure 1**. The Device Toolbar
-
-By default the Device Toolbar opens in Responsive Viewport Mode.
+By default, the Device Toolbar opens in viewport with **Dimentions** set to **Responsive**.
```
Comment (rachelandrew) [inline site/en/docs/devtools/device-mode/index.md:46]:
```suggestion
By default, the Device Toolbar opens in viewport with **Dimensions** set to **Responsive**.
```

File: site/en/docs/devtools/device-mode/index.md [hunk 2]
```diff
@@ -88,33 +80,29 @@ of cursor you see when you hover over the page. **Events fired** refers to wheth
 
 ### Mobile Device Viewport Mode {: #device }
 
-To simulate the dimensions of a specific mobile device, select the device from the **Device** list.
+To simulate the dimensions of a specific mobile device, select the device from the **Dimensions** list.
 
-{% Img src="image/admin/q5MGmsuKDvz67EWNfriE.png", alt="The Device list.", width="800", height="500" %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/2JDEYmYANUwYnNUrbe1e.png", alt="The Dimentions list.", width="800", height="507" %}
```
Comment (rachelandrew) [inline site/en/docs/devtools/device-mode/index.md:85]:
```suggestion
{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/2JDEYmYANUwYnNUrbe1e.png", alt="The Dimensions list.", width="800", height="507" %}
```

File: site/en/docs/devtools/device-mode/index.md [hunk 3]
```diff
@@ -88,33 +80,29 @@ of cursor you see when you hover over the page. **Events fired** refers to wheth
...
 
-**Figure 6**. The Device list
+For more information, see [Add a custom mobile device](#cusom).
```
Comment (rachelandrew) [inline site/en/docs/devtools/device-mode/index.md:87]:
```suggestion
For more information, see [Add a custom mobile device](#custom).
```

File: site/en/docs/devtools/device-mode/index.md [hunk 4]
```diff
@@ -123,156 +111,118 @@ just doesn't have art for that specific option.
...
+1.  If you're adding your own, enter a name, width, and height for the device, then click **Add**.
 
-    {% Img src="image/admin/on0B3hv1k23NRXjE1TzW.png", alt="Creating a custom device.", width="800", height="548" %}
+    {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/BrFpL7MQ0VYzl1P1rtC6.png", alt="Creating a custom device.", width="800", height="681" %}
 
-    **Figure 12**. Creating a custom device
+    The [device pixel ratio][9], [user agent string][10], and [device type][11] fields are optional. The device type field is the list that is set to **Mobile** by default.
 
-### Show rulers {: #rulers }
+1. Back in the viewport, select the newly added device from the **Dimentions** list.
```
Comment (rachelandrew) [inline site/en/docs/devtools/device-mode/index.md:134]:
```suggestion
1. Back in the viewport, select the newly added device from the **Dimensions** list.
```

File: site/en/docs/devtools/device-mode/index.md [hunk 5]
```diff
@@ -37,47 +38,38 @@ actually runs on a mobile device.
 ## Simulate a mobile viewport {: #viewport }
 
 Click **Toggle Device Toolbar**
-{% Img src="image/admin/9FiBHFCzfPgP8sy6LMx7.png", alt="Toggle Device Toolbar", width="30", height="32" %} to open
+{% Img src="image/admin/9FiBHFCzfPgP8sy6LMx7.png", alt="Toggle Device Toolbar", width="20", height="22" %} to open
```
Comment (rachelandrew) [inline site/en/docs/devtools/device-mode/index.md:41]:
```suggestion
{% Img src="image/admin/9FiBHFCzfPgP8sy6LMx7.png", alt="Toggle Device Toolbar.", width="20", height="22" %} to open
```

**Review by rachelandrew — APPROVED** [review]
Just a few nits.

**Review by jecfish — APPROVED** [review]

**Pull request merged**